components/inputs/input-date.js

```diff
@@ -276,7 +268,8 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 					hide-invalid-icon
 					id="${this._inputId}"
 					label="${ifDefined(this.label)}"
-					?label-hidden="${this.labelHidden}"
+					?label-hidden="${this.labelHidden || this.labelledBy}"
+					.labelRequired="${false}"
```

Comment:
When a component passes the label down to another component that also uses `LabelledMixin`, we need to turn off the label validation for the nested component. Otherwise when there's an error, a missing label exception will get thrown from both the top-level and lower-level component.

Reply:
Would this be needed in `input-date-time` then since it does pass a label down to the `input-date` and `input-time` components?

Reply:
Indeed it would -- thanks! I did `input-date` before I discovered this problem and then didn't circle back. Fixed now.